go/pkg/pwapi/api_team-send-invite.go

```diff
@@ -55,23 +55,17 @@ func (svc *service) TeamSendInvite(ctx context.Context, in *TeamSendInvite_Input
 		return nil, errcode.ErrNotTeamOwner.Wrap(err)
 	}
 
-	// check that the invited user is not already in this season
-	var existCheck pwdb.TeamMember
+	// check if invited user already has a team in this season
+	var existingSeasonMembership pwdb.TeamMember
 	err = svc.db.
-		Where(pwdb.TeamMember{
-			UserID: invitedUserID,
-		}).
-		Where(&pwdb.Season{
-			ID: team.SeasonID,
-		}).
-		Joins("JOIN team ON team_member.team_id = team.id").
-		Joins("JOIN season ON team.season_id = season.id").
-		First(&existCheck).
+		Model(pwdb.TeamMember{}).
+		Joins("JOIN team on team.id = team_member.team_id AND team.season_id = ? AND team.deletion_status = ?", team.SeasonID, pwdb.DeletionStatus_Active).
+		Preload("Team").
```

Comment:
```suggestion
		Preload("Team").
```

from what I can see, you don't use the variable, so preloading is useless and makes an unecessary request

btw in general you should consider using the `Count()` to just check that there are results